Dockerfiles/mlpack-dependencies-debian/Dockerfile

```diff
@@ -0,0 +1,58 @@
+#Using debian's latest image as base-image
+FROM debian:latest
+
+RUN apt-get update  && apt-get install -y aptitude && apt-get purge -y \
+    $(aptitude search '~i!~M!~prequired!~pimportant!~R~prequired! \
+    ~R~R~prequired!~R~pimportant!~R~R~pimportant!busybox!grub!initramfs-tools' \
+    | awk '{print $2}') && apt-get purge -y aptitude && \
+    apt-get autoremove -y && apt-get clean && rm -rf /usr/share/man/?? && \
+    rm -rf /usr/share/man/??_*
+
+#Installing dependencies required to run mlpack
+RUN apt-get update  && apt-get install -y --no-install-recommends wget \
+    libboost-math-dev libboost-program-options-dev libboost-test-dev \
+    libboost-serialization-dev cmake g++ binutils-dev make g++ txt2man git \
+    doxygen liblapack-dev libblas-dev libarpack2 libsuperlu-dev  && \
+    apt-get clean && rm -rf /usr/share/man/?? && rm -rf /usr/share/man/??_* && \
+    rm -rf /var/lib/apt/lists/* && rm -rf /usr/share/locale/* && \
+    rm -rf /var/cache/debconf/*-old && rm -rf /usr/share/doc/*
+
+#Installing armadillo via source-code
+ADD armadillo-code /armadillo-code
+WORKDIR armadillo-code
+RUN cmake .
+RUN make
+RUN make install
+
+#Removing setup files
+RUN rm -rf /armadillo-code
+
+#creating a non-root user
+RUN adduser --system --disabled-password --disabled-login \
+    --shell /bin/sh mlpack
+
+#Hardening the containers by unsetting all SUID tags
+RUN for i in `find / -perm 6000 -type f`; do chmod a-s $i; done
+
+#Changing work directory 
+WORKDIR /home/mlpack/
+
+#Setting up non-root user
```

Comment:
This is a picky style comment, but could you use a space between the `#` and the comment like the rest of the mlpack codebase please?  If you can also make the comments complete sentences (which I think in this case is pretty much just adding a period to the end), then it will conform to the mlpack style guide: https://github.com/mlpack/mlpack/wiki/DesignGuidelines#style-guidelines